setup.md

```diff
@@ -3,6 +3,52 @@ layout: page
 title: Setup
 ---
 
+## Getting ready
 
+You need to install MarcEdit and download a data file to follow this lesson.
 
-{% include links.md %}
+### Installing and running MarcEdit
+
+MarcEdit is a free, open-source .NET application. You can download MarcEdit from 
+[https://marcedit.reeset.net/downloads](https://marcedit.reeset.net/downloads). 
+This lesson has been tested with the version or MarcEdit 7 for Windows and MarcEdit MacOS 3 up to the latest version of 7.5.21.153 for MarcEdit 7 for Windows.
+
+Packages are available on <https://marcedit.reeset.net/downloads> for Windows, MacOS, and Linux. System requirements for the current release are available on <https://marcedit.reeset.net/downloads>.
+
+Please download the latest stable version, choosing the "Windows Build Options" or MacOS.
+
+If choosing the "Windows Build Options", please read through the pop up instructions that explain the differences between a self contained and consolidated installation. The majority of people will choose the self contained installation or a download for one user on a computer in contrast to downloading MarcEdit on a computer accessed by many users. To download the user only installation, choose "Download MarcEdit 7.5 (All Users)".
+
+Installation video demostrations are available for both types of installations: self contained <https://youtu.be/FX0r7jIxeN4>, and consolidated <https://youtu.be/7wYo7VoOwMI>. 
+
+Instructions for system administrators has been provided. Also information on managing MarcEdit installations are available <https://marcedit.reeset.net/managing-marcedit-installations>.
+
+If you are using an older version of MarcEdit, it is recommended you upgrade to the latest tested version. 
+
+Notes:
+* The cosolidating build is the only one that currently supports 32-bit systems.
+* Currently Linux is not supported for version 7.5x only 7.3x for Windows and MacOS 3.2x. 
+* If downloading for Linux, the current Mono is required for installation.
+* When downloading MarcEdit for the first time, the setup wizard will appear. Remember to go through the wizard to set up the applicaiton configrations.
```

Comment:
Do they require more guidance for setting up configurations? Should this be moved to the top of the list?